kivy/input/providers/probesysfs.py

```diff
@@ -82,7 +82,12 @@ def name(self):
 
         def get_capabilities(self):
             path = os.path.join(self.path, "device", "capabilities", "abs")
-            line = read_line(path)
+            line = "0"
+            try:
+                line = read_line(path)
+            except OSError:
```

Comment:
Is this not throwing a `IOError` (python2) or `FileNotFoundError` (python3) instead?

Reply:
I think it returns OSError for Windows, but I'm not sure if OSError is the parent or IOError, sooo... test or check with cpython source I'd say :) It should handle FNF in either case for py3 iirc.

Reply:
@AndreMiras This resolves FNF → https://github.com/python/cpython/blob/621cebe81b1e6c8de10425955bf532d31ee4df42/Objects/exceptions.c#L1256 (`OSError`), for the rest just append IOError if it still brags :)

Reply:
Oh didn't know about that thingy. Thanks